tests/test_tutorials.py

```diff
@@ -8,6 +8,8 @@
 
 from conftest import here
 
+skip_py2 = True if sys.version_info[0] == 2 else False
```

Comment:
This can simply be:

```
skip_py2 = sys.version_info[0] == 2
```

and would suggest renaming to `is_py2` or similar

Reply:
done